### DISCUSSION TITLE
Unable to setup project in development

### DISCUSSION THREAD

**Discussion by Xiddiqui** [LiveHelperChat/livehelperchat#2056] [Q&A] [answered]
I am facing issues in setting up project in development mode . I need help . I am following this guide https://doc.livehelperchat.com/docs/development/quick-guide . I am attaching error screen shot . Kindly guide me how can i run project in development . I have installed webpack too but still facing this error . I have attached screen shot of package.json you can see its there
<img width="501" alt="Screenshot 2024-02-23 at 10 14 19 AM" src="https://github.com/LiveHelperChat/livehelperchat/assets/70687212/d75ca834-af9e-4838-8585-de677cf8ae4f">

<img width="633" alt="Screenshot 2024-02-23 at 10 06 57 AM" src="https://github.com/LiveHelperChat/livehelperchat/assets/70687212/531a5960-870d-448e-bedd-a9d4b7373059">

**Comment by remdex** [comment]
I ques you will have to educate yourself more how to work with nodejs and it's dependencies. You should first install packages using `npm install` and then try todo everything else...

**Reply by AbdullahMoqeet** [reply]
I first run npm install thats the basic thing i know . After that when i try to run it gives above error

**Reply by remdex** [reply]
Well errors tells it all... No much I can gelp there. Perhaps just to old node version... And if I see from screenshot it's for reach version. It's enough there to run just
```
npm install
npm run build
```

**Reply by AbdullahMoqeet** [reply]
okay Thank you for your support. I tried on node 18 , 14 and 12  facing same issue .

**Reply by remdex** [reply, marked as answer]
well, well gain more knowledge and make it work :) You can also delete package.lock file and then do install buil thing :)
